src/test/python/test_VantiqSDKMock.py

```diff
@@ -532,18 +532,18 @@ async def check_crud_operations(self, mocked, client: Vantiq, skip_pretest_clean
 
         mocked.get(f'/api/v1/resources/k8sclusters/foo', status=404,
                    body=json.dumps({'code': 'io.vantiq.resource.not.found',
-                                    'message': "The requested instance ('[name:foo]') of the k8sclusters resource"
+                                    'message': "The requested instance ('{name=foo}') of the k8sclusters resource"
                                                " could not be found.",
-                                    'params': ['k8sclusters', '[name:foo]']}))
+                                    'params': ['k8sclusters', '{name=foo}']}))
         vr = await client.select_one(VantiqResources.K8S_CLUSTERS, 'foo')
         assert isinstance(vr, VantiqResponse)
         assert not vr.is_success
         errs = vr.errors
         assert isinstance(errs, list)
         ve = errs[0]
-        assert ve.message == "The requested instance ('[name:foo]') of the k8sclusters resource could not be found."
+        assert ve.message == "The requested instance ('{name=foo}') of the k8sclusters resource could not be found."
         assert ve.code == 'io.vantiq.resource.not.found'
-        assert ve.params == ['k8sclusters', '[name:foo]']
+        assert ve.params == ['k8sclusters', '{name=foo}']
```

Comment:
Is this a Vantiq formatting change?

Reply:
yes, looks like formatting changed -- these allow the tests to pass when run against a real server.